api.rst

```diff
@@ -1452,6 +1452,68 @@ You can use a tool like `Swagger UI <https://swagger.io/tools/swagger-ui/>`_ to
+
+CORS
+----
+
+PostgREST sets highly permissive cross origin resource sharing, that is why it accepts Ajax requests from any domain.
+
+Tables
+------
+
+Suppose you make this API request to an existing table named ``people``.
+
+.. code-block:: http
+
+  OPTIONS /people HTTP/1.1
+
+Then, the response you get is:
+
+.. code-block:: http
+
+  HTTP/1.1 200 OK
+  Allow: OPTIONS,GET,HEAD,POST,PUT,PATCH,DELETE
+
+
+Views
+-----
+
+The methods that can be used on views are determined by the presence of INSTEAD OF TRIGGERS and primary key columns:
+
+.. table::
+   :widths: auto
+
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | Method allowed     | View's requirements                                                                             |
+   +====================+=================================================================================================+
+   | OPTIONS, GET, HEAD | None (Always allowed)                                                                           |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | POST               | INSTEAD OF INSERT TRIGGER                                                                       |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | PUT                | INSTEAD OF INSERT TRIGGER, INSTEAD OF UPDATE TRIGGER, primary key columns                       |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | PATCH              | INSTEAD OF UPDATE TRIGGER                                                                       |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | DELETE             | INSTEAD OF DELETE TRIGGER                                                                       |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+   | All the above methods are allowed for                                                                                |
+   | `auto-updatable views <https://www.postgresql.org/docs/current/sql-createview.html#SQL-CREATEVIEW-UPDATABLE-VIEWS>`_ |
+   +--------------------+-------------------------------------------------------------------------------------------------+
+
+Functions
+~~~~~~~~~
+
+For database function endpoints, OPTIONS requests are not supported.
+
+.. important::
+  Whenever you add or remove tables or views, or modify a view's definition, primary key columns or INSTEAD OF TRIGGERS on the database, you must refresh PostgREST's schema cache for OPTIONS requests to work properly. See the section :ref:`schema_reloading`.
```

Comment:
Summarizing this a bit, since it's mostly about views
```suggestion
  Whenever you modify a view's INSTEAD OF TRIGGERS on the database, you must refresh PostgREST's schema cache for OPTIONS requests to work properly. See the section :ref:`schema_reloading`.
```

Reply:
> Summarizing this a bit, since it's mostly about views

I think at least a mention to table creation should be kept there. If a new table is created the OPTIONS result will be a 404 till the schema is reloaded, for instance.

Reply:
Cool, I agree. A mention to tables' existence should be kept.